packages/components/tests/dummy/app/templates/components/form/base-elements.hbs

```diff
@@ -50,6 +50,16 @@
         <code class="dummy-code">&lt;label&gt;</code>
         element.</p>
     </dd>
+    <dt>isRequired <code>boolean</code></dt>
+    <dd>
+      <p>It appends a badge to the label text to indicate user input is required.</p>
+      <p>Default: <span class="default">false</span></p>
+    </dd>
+    <dt>isOptional <code>boolean</code></dt>
+    <dd>
+      <p>It appends "(Optional)" to the label text to indicate user input is not required.</p>
+      <p>Default: <span class="default">false</span></p>
+    </dd>
```

Comment:
[nitpick] Probably I would generalize the text, so that we don't have to specify there is some kind of difference between`badge` and `("Optional")` (it's kinda an internal thing).  Something like this:
- It appends a [mark|indication] to the label text to indicate that the user input is required.
- It appends a [mark|indication] to the label text to indicate that the user input is optional.